rasa_sdk/forms.py

```diff
@@ -265,7 +265,7 @@ def extract_other_slots(
                     should_fill_entity_slot = (
```

Comment:
The comment above this should i guess be sth like "check whether the slot should be filled by an entity in the input"